apps/breethe/lib/breethe/sources/eea.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Breethe.Sources.EEA do
+  alias __MODULE__.CSV
+
+  def country_codes(),
+    do: [
+      "BG",
+      "CY",
+      "CZ",
+      "ES",
+      "GR",
+      "HR",
+      "HU",
+      "IE",
+      "IS",
+      "IT",
+      "LT",
+      "LU",
+      "NO",
+      "PT",
+      "RS",
+      "SE",
+      "SK",
+      "AD",
+      "BA",
+      "BE",
+      "DE",
+      "FI",
+      "FR",
+      "GB",
+      "GI",
+      "LV",
+      "MT",
+      "NL",
+      "SI",
+      "DK",
+      "GE",
+      "MK",
+      "PL",
+      "AT",
+      "CH",
+      "EE"
+    ]
+
+  # @parameters ["PM10", "PM2.5", "SO2", "NO2", "O3", "CO"]
+  def parameters(), do: ["SO2", "NO2", "O3", "CO"]
+
+  def get_data() do
+    for country <- country_codes(), param <- parameters() do
+      country
```

Comment:
Can we run each one in its own `Task`? Or maybe use https://github.com/dashbitco/broadway ?

Reply:
Yep, my intention was to implement this in a follow-up (also because I'll be refactoring a few things including how we manage asnyc tasks for requests)?